packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -147,6 +147,34 @@
         <protected>false</protected>
         <shortDescription>Next</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_PreviousPage</fullName>
+        <value>Previous Page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Previous Page</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_NextPage</fullName>
+        <value>Next Page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Next Page</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_GoToPage</fullName>
+        <value>Go to page {{0}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Go to page ...</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowNResultsPerPage</fullName>
+        <value>Show {{0}} per page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Show N number of results per page</shortDescription>
```

Comment:
Your value and shortDescription don't exactly match right?

1 is `show 5 per page` while the description says : `show 5 number of results per page` ?

If possible we should be saying `show 5 results per page`